src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
```

Comment:
That only works in some of the cases and it is the reason this is currently not broken but it isn't the correct way to configure. The correct way to configure this is to set the OSGroup in the builds file. 

The only real purpose for having the configuration defatuled to Windows_Debug in the csproj is to enable a default when building the csproj directly from the command line.